src/network/networkd-link.c

```diff
@@ -721,10 +721,15 @@ void link_check_ready(Link *link) {
                     !link->ipv4ll_route)
                         return;
 
-        if (link_ipv6ll_enabled(link))
+        if (link_ipv6_enabled(link)) {
+
                 if (in_addr_is_null(AF_INET6, (const union in_addr_union*) &link->ipv6ll_address) > 0)
                         return;
 
+                if (link_ipv6_accept_ra_enabled(link) && !link->ndisc_configured)
```

Comment:
Right, and I'm saying that 2b00a4e is conceptually wrong: there are cases where we want bridge slaves to not get an IP, but other cases where we do want that (anonymous bridges). So 2b00a4e is already wrong, and the reason why your PR here makes things worse because it now calls `link_ipv6_enabled()` which does that unconditional `if (link->network->bridge) return false`. I. e. I'm not saying that your PR here is wrong by itself, but that whole bridge handling needs to be fixed.

>what you mean by this actually ?
>    Bridge slave don't get a IPv6 address.

Sorry, I'm not sure I understand this?

It seems we should maybe discuss this on IRC or a video chat? It seems we keep talking past each other.